src/interface/widgets/widget.cpp

```diff
@@ -233,7 +233,7 @@ void Visualizer::drawFieldHints(const Field &field, QPainter &painter) {
     pen.setColor(Qt::gray);
     painter.setPen(pen);
 
-    auto lineStart = toScreenPosition(Vector2(field.getOurGoalCenter().x, lineY));
+    auto lineStart = toScreenPosition(Vector2(field.getLeftmostX(), lineY));
     auto lineEnd = toScreenPosition(Vector2(0, lineY));
```

Comment:
I'd prefer seeing this:

```suggestion
    auto lineStart = toScreenPosition({ field.getLeftmostX(), lineY });
    auto lineEnd = toScreenPosition({ 0, lineY });
```

Reply:
I actually think the explicit Vector2 call makes it more clear